Sources/SourceKitBazelBSP/RequestHandlers/SKOptions/CompilerArgumentsProcessor.swift

```diff
@@ -266,3 +209,14 @@ enum CompilerArgumentsProcessor {
         lines[idx + 1] = new
     }
 }
+
+enum CompilerArgError: Error, CustomStringConvertible {
+    case targetNotFound(String)
+    case actionNotFound(String)
+    var description: String {
+        switch self {
+        case .targetNotFound(let target): "Target: \(target) not found."
+        case .actionNotFound(let target): "Action not found for target: \(target)."
+        }
+    }
+}
```

Comment:
If we go with my other suggestion to print the errors directly to better please OSLog, then we don't need this bit